R/unit.conversions.R

```diff
@@ -46,6 +46,10 @@ NULL
 #' @keywords data units internal
 generateUnitsData <- function() {
   # valid.metadata.units
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  numerator=denominator=value=NULL
```

Comment:
move these lines closer to where the need for them occurs, please. right after line 132, `# Append the reverse conversions`, would be appropriate.